DEVELOPMENT.md

```diff
@@ -35,45 +35,54 @@ Install these tools:
     ```bash
     kind create cluster
     ```
+   **NOTE**: The KinD cluster can also be created with tools like 
+   [Podman Desktop](https://podman-desktop.io/docs/kind/creating-a-kind-cluster).
+ 
 
 1. Set up `ko` to upload images to the KinD cluster, or any other registry you
   want to use:
     ```bash
     export KO_DOCKER_REPO="kind.local"
     ```
+   **NOTE**: if your cluster doesn't have the name `kind` you need to specify the env var `KIND_CLUSTER_NAME`.
 
 ## Install KubeArchive
 
 1. Use Helm to install KubeArchive:
-    ```bash
-    helm install kubearchive charts/kubearchive \
-        --set-string apiServer.image=$(ko build github.com/kubearchive/kubearchive/cmd/api) \
-        --set-string sink.image=$(ko build github.com/kubearchive/kubearchive/cmd/sink)
-    ```
+   ```bash
+   helm install kubearchive charts/kubearchive -n default \
+       --set-string apiServer.image=$(ko build github.com/kubearchive/kubearchive/cmd/api) \
+       --set-string sink.image=$(ko build github.com/kubearchive/kubearchive/cmd/sink)
+   ```
 1. Check that the KubeArchive deployments are Ready:
-    ```bash
-    kubectl get -n kubearchive deployments
-    ```
+   ```bash
+   kubectl get -n kubearchive deployments
+   ```
+   
+1. List the deployed helm chart
+   ```bash
+   helm list -n default  
+   ```
```

Comment:
You are autoindenting on three spaces, we can create an .editorconfig to prevent these inconsistencies.

Reply:
yes, my guess is that my editor is autoaligning the code blocks with the start of the paragraph. If we want to enforce it to a 4 space indent, it's better to set that in a `.editorconfig` file. I will check that out.

Reply:
I created #57 to address this, I assigned it to you, feel free to unassign it if you don't want to work on it.